rqt_gui/setup.py

```diff
@@ -0,0 +1,38 @@
+from setuptools import setup
+
+package_name = 'rqt_gui'
+setup(
+    name=package_name,
+    version='0.5.0',
+    package_dir={'': 'src'},
+    packages=['rqt_gui'],
+    data_files=[
+        ('share/ament_index/resource_index/packages',
```

Comment:
Why does this work on Windows?? Shouldn't it be `share\ament_index\...`?

Reply:
setuptools takes care of the cross-platform bit of this code.